CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project's packages adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [v0.8.0] WIP
+## [v0.9.0] WIP
```

Comment:
Let's have a changelog entry for the migration.

Reply:
Thanks, I forgot that on other repos.